### PR TITLE
NAT Gateway Template

### PR DESCRIPTION
**Pull request by sourced-vince**

### CODE REVIEW

File: community/cloud-foundation/templates/nat_gateway/nat_gateway.py [hunk 1]
```diff
@@ -11,10 +11,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""This template creates an HA NAT gateway."""
+""" This template creates an HA NAT gateway. """
 
 
-SETUP_NATGATEWAY_SH = """#!/bin/bash
+SETUP_NATGATEWAY_SH = """ #!/bin/bash
```
Comment (sourced-vince) [inline community/cloud-foundation/templates/nat_gateway/nat_gateway.py]: SETUP_NATGATEWAY_SH is a variable to be used in the template. Please revert the changes you made to it. Its actual code.

Comment (alexander-levitski) [reply community/cloud-foundation/templates/nat_gateway/nat_gateway.py]: In a minute...
